app/views/includes/back-to-choices.nunjucks

```diff
@@ -1,6 +1,9 @@
 <div class="notification-banner notification-banner__hiviz js-feedback-banner">
   <p class="notification-banner--inner">
-    This is a new service. Go <a href="http://www.nhs.uk/">back to NHS Choices</a>.
+    {% if gp.oldProfileLink %}
```

Comment:
This makes it looks like `oldProfileLink` might be able to be falsy (which it can't be). The check should be for the entire `gp` object as that can be falsy on some pages e.g. the 404 page. I suggest this is changed to check just for `gp`.